gef-ui/server/controllers/select-exporters-correspondence-address/index.js

```diff
@@ -27,7 +27,7 @@ const validateSelectExportersCorrespondenceAddress = (req, res) => {
   const parseAddresses = JSON.parse(addresses);
   let selectedAddressError;
 
-  if (isEmpty(selectedAddress)) {
+  if (isEmpty(selectedAddress) || selectedAddress.includes('Addresses Found')) {
```

Comment:
This is required, otherwise the validation thinks that the first select option (with "X Addresses Found" is a valid option.

Reply:
Thats a good shout as a fix -- 

Trying not to get nerd sniped by this (And I don't think we should for the sake of this ticket) -- but I think there's probably a neater way of doing this using `disabled` if we can override the first element (if this is a drop down) ie `<option selected disabled>10 Addresses Found</option>` would disable this option from ever getting sent. Could probably pass in `:first-child disabled` or some sort of custom style like that.

Side note - `:user-valid` is a new CSS class which is cool for text input fields that allows for a similar thing on text fields

Reply:
Opps -- I resolved this comment thinking it was mine, happy with this to be resolved whenever you are